src/app/services/blockchain.service.ts

```diff
@@ -66,7 +68,7 @@ export class BlockchainService {
       .filter(status => !!status)
```

Comment:
this filter should probably be removed as the status is no longer related to connections but version